gtk/src/lib.rs

```diff
@@ -16,71 +16,71 @@
 //!
 //! See also:
 //!
-//! - [Gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
+//! - [gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
 //!
 //! - [General `GLib` family types and object system overview](../glib/index.html)
 //!
-//! - [GTK+ documentation](https://www.gtk.org/docs/)
+//! - [GTK documentation](https://www.gtk.org/docs/)
 //!
-//! # Hello World
+//! ## “Hello, World!” example program
 //!
 //! ```no_run
 //! use gtk::prelude::*;
-//! use gtk::{ButtonsType, DialogFlags, MessageType, MessageDialog, Window};
+//! use gtk::{ButtonsType, MessageDialogBuilder};
 //!
 //! fn main() {
-//!     if gtk::init().is_err() {
-//!         println!("Failed to initialize GTK.");
-//!         return;
-//!     }
-//!     MessageDialog::new(None::<&Window>,
-//!                        DialogFlags::empty(),
-//!                        MessageType::Info,
-//!                        ButtonsType::Ok,
-//!                        "Hello World").run();
+//!     gtk::init().expect("Failed to initialize GTK.");
+//!
+//!     MessageDialogBuilder::new()
+//!         .text("Hello, World!")
+//!         .buttons(ButtonsType::Close)
+//!         .build()
+//!         .run();
```

Comment:
Should use show here

Reply:
Should maybe be something more useful here like the Hello World on https://gtk.org (which is about the same number of lines...)

Reply:
The example would not work with show because there is no main loop. If you think it's worth it, I can restructure the examples. For gtk4 I would just give one example and mostly link to the book for the rest.

Reply:
I will just drop this example, there is a proper example.

Reply:
> The example would not work with show because there is no main loop

We should replace that with something that the user can copy/paste and have something working.

Reply:
Yeah dropping the example and just keeping the one below makes most sense, I guess (and maybe replacing the example below with the gtk.org one).